.github/workflows/meta_tests.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         with:
           # TODO: Since pytype is not available for Python 3.13, and
           # pytype_test.py imports pytype, we need to use Python 3.12 for now.
-          python-version: "3.12"
+          python-version: "3.13"
```

Comment:
I think it installs now, but there's still no prebuilt wheels
https://github.com/google/pytype/issues/1877